packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -802,6 +802,7 @@ export type CallCompositeOptions = {
     joinCallOptions?: {
         microphoneCheck?: 'requireMicrophoneAvailable' | 'skip';
     };
+    hideLocalScreenShareStream?: boolean;
```

Comment:
can you move this inside the existing `galleryOptions`?

Reply:
Avoid booleans, change to something like `localScreenshareView: { 'stream' | 'placeholder' }` (open to better naming!)

Reply:
Customers understand what the default behaviour is and understand they want to be a little special so they are willing to do this additional change if they want it. It's ok to not have this API on the top level. This is a good discussion as we should be thinking where should these APIs exist.